python/foglamp/services/south/modify_process.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+"""Modify process_name for Python Async plugins from south to south_c"""
+
+import aiohttp
+import asyncio
+
+from foglamp.common import logger
+from foglamp.common.storage_client.exceptions import StorageServerError
+from foglamp.common.storage_client.payload_builder import PayloadBuilder
+from foglamp.common.storage_client.storage_client import StorageClientAsync
+from foglamp.common.configuration_manager import ConfigurationManager
+
+__author__ = "Amarendra K Sinha"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+_LOGGER = logger.setup(__name__)
+_HTTP = 'http'
+_HOST = 'localhost'
+_PORT = '8081'
+
+
+async def disable_service(service_name):
+    # Disable schedule
+    try:
+        put_url = "{}://{}:{}/foglamp/schedule/disable".format(_HTTP, _HOST, _PORT)
+        data = '{"schedule_name": "%s"}' % service_name
```

Comment:
use `.format` ; `%s` is old style and only good for logging

Reply:
.format uses {var-name} as placeholder for var. Hence % style has been used here.